internal/clients/sources/sources_real.go

```diff
@@ -0,0 +1,12 @@
+package sources
+
+import (
+	"context"
+)
+
+var GetSourcesClient func(ctx context.Context) (SourcesIntegration, error)
+
+type SourcesIntegration interface {
+	ShowSourceWithResponse(ctx context.Context, id ID, reqEditors ...RequestEditorFn) (*ShowSourceResponse, error)
+	ListSourcesWithResponse(ctx context.Context, params *ListSourcesParams, reqEditors ...RequestEditorFn) (*ListSourcesResponse, error)
+}
```

Comment:
Sorry this should have been `sources_interface.go`.

Reply:
One detail: We use RequestEditorFn interface from the generated code, ideally we should abstract that.